--- stock_kernel/linux-5.19.8/arch/x86/entry/syscalls/syscall_64.tbl	2022-11-13 02:22:44.079957260 +0530
+++ edited_kernel/linux-5.19.8/arch/x86/entry/syscalls/syscall_64.tbl	2022-11-14 04:21:32.883358031 +0530
@@ -372,6 +372,7 @@
 448	common	process_mrelease	sys_process_mrelease
 449	common	futex_waitv		sys_futex_waitv
 450	common	set_mempolicy_home_node	sys_set_mempolicy_home_node
+451	common	kernel_2d_memcpy	sys_kernel_2d_memcpy
 
 #
 # Due to a historical design error, certain syscalls are numbered differently
